src/System.Text.Json/tests/Utf8JsonReaderTests.cs

```diff
@@ -1770,66 +1770,89 @@ public static void ConsumeLotsOfComments(string valueString, bool insideArray, s
             Assert.Equal(dataUtf8.Length, json.BytesConsumed);
         }
 
+        struct SingleCommentVerifyInfo
+        {
+            public bool startObjectFound;
+            public bool endObjectFound;
+            public bool commentFound;
+        }
+
+        private static void VerifyReadLoop(ref Utf8JsonReader json, string expected, ref SingleCommentVerifyInfo verifyInfo)
+        {
+            while (json.Read())
+            {
+                switch (json.TokenType)
+                {
+                    case JsonTokenType.StartObject:
+                        verifyInfo.startObjectFound = true;
```

Comment:
Is this necessary? I don't really see verifying that we saw start/end object add value to the test here since we know it always will be true given the input (especially since the input/test is related specifically to comments).